test/util/Jamfile.v2

```diff
@@ -13,13 +14,47 @@
 # Version 1.0. (See accompanying file LICENSE_1_0.txt or copy at
 # http://www.boost.org/LICENSE_1_0.txt)
 
-test-suite boost-geometry-util
-    :
-    [ run calculation_type.cpp ]
-    [ run for_each_coordinate.cpp ]
-    [ run math_sqrt.cpp ]
-    [ run range.cpp ]
-    [ run rational.cpp ]
-    [ run select_most_precise.cpp ]
-    [ run write_dsv.cpp ]
+# bring in rules for testing
+import testing ;
+
+project boost/geometry/util/test
+    : requirements
+      <source>/boost/test//boost_unit_test_framework
+      <define>BOOST_GEOMETRY_TEST_BUILD_WITH_B2
```

Comment:
Sorry, all this cannnot be accepted.
Maybe it is a good idea to first ask on the list if this is a good idea, and then submit a PR. That avoids creating the code, the PR and large discussions